okhttp/src/main/java/io/grpc/okhttp/OutboundFlowController.java

```diff
@@ -46,9 +46,15 @@
   }
 
   /**
-   * Must be called with holding transport lock.
+   * Adjusts outbound window size requested by peer. When window size is increased, it does not send
+   * any pending frames. Caller should send any pending frames after notify the peer about
```

Comment:
s/Caller should send any pending frames/If this method returns `true`, the caller should call `writeStreams()`/